### DISCUSSION TITLE
Subdomain for vercel app project

### DISCUSSION THREAD

**Discussion by xscottxbrownx** [vercel/community#1955] [Help] [answered]
### Summary

I read through the docs, and thought I have followed everything fine - but seem to still have issues getting my deployed vercel app to show "fmc.scottxbrown.com" in URL address.

I added the CNAME to my provider:

<img width="1423" alt="Screenshot 2023-04-03 at 11 34 11 AM" src="https://user-images.githubusercontent.com/71395931/229598413-251769ff-c23c-4120-a6ef-7fc8967a6e8b.png">

Receive error in vercel dashboard:

<img width="960" alt="Screenshot 2023-04-03 at 11 34 19 AM" src="https://user-images.githubusercontent.com/71395931/229598427-9c39ca9a-d2c2-4a51-ab48-539fec7650d0.png">

**Comment by TooTallNate** [comment, marked as answer]
I'm not seeing that DNS record propagated anywhere in the world so far: https://www.whatsmydns.net/#CNAME/fmc.scottxbrown.com

This would indicate an issue/misconfiguration on the DreamHost side.

**Reply by xscottxbrownx** [reply]
could you expand on that a little?   I did nothing but add that CNAME like vercel directed.

For now I just have fmc.scottxbrown.com redirecting to the vercel given url.

**Reply by TooTallNate** [reply]
It looks correct based on the screenshot you provided, but I'm not sure if DreamHost maybe has some additional steps required to "commit" the change. I suggest contacting DreamHost support.
